packages/react-percy/src/cli.js

```diff
@@ -11,7 +11,7 @@ const VERSION = require('../package.json').version;
 
 function formatError(error) {
   if (error.error && error.error.errors && error.error.errors.length) {
-    return error.error.errors.map(e => e.datail).join(', ');
+    return error.error.errors.map(e => e.detail).join(', ');
```

Comment:
Fixing a dumb typo so you can actually see error messages from Percy API calls:

![image](https://user-images.githubusercontent.com/733696/30253965-21c52a0c-965e-11e7-919a-175d29e46afe.png)